charts/zabbix/templates/job-init-db-schema.yaml

```diff
@@ -35,6 +35,6 @@ spec:
         command:
           - "/bin/sh"
           - "-c"
-          - 'sed -e "s/^exec \"\$@\"$/prepare_server/" -e "/^ *update_zbx_config$/d" /usr/bin/docker-entrypoint.sh >~/docker-entrypoint.sh && bash ~/docker-entrypoint.sh'
+          - 'export V=$(zabbix_server -V | head -n 1 | sed "s/.*\([0-9]\+\.[0-9]\+\.[0-9]\+\)/\1/"); if [ "$V" = "`echo -e "$V\n6.2.6" | sort -V | head -n1`" ]; then sed -e "s/^exec \"\$@\"$/prepare_server/" -e "/^ *update_zbx_config$/d" /usr/bin/docker-entrypoint.sh >~/docker-entrypoint.sh && bash ~/docker-entrypoint.sh; else /usr/bin/docker-entrypoint.sh init_db_only; fi'
```

Comment:
Won't this break all 6.2.0-5 versions?

Reply:
6.0 is not affected?

Reply:
I don't get it. Why should it break these?

```
alias vercheck='if [ "$V" = "`echo -e "$V\n6.2.6" | sort -V | head -n1`" ]; then echo old; else echo new; fi'

export V=6.2.0; vercheck
old

export V=6.2.5; vercheck
old

export V=4.4.13; vercheck
old

export V=6.2.6; vercheck
old

export V=6.2.7; vercheck
new

export V=6.2.8; vercheck
new

export V=6.4.0; vercheck
new
```

I suppose that every version newer than 6.2.7 will also have this new "feature". Once we don't have to support any old (LTS) versions anymore (6.0...) this entire job manifest can be changed to not supply any command but only an argument `init_db_only`. But that's gonna take some while.

Reply:
Sorry, misinterpreted your bash, everything is great.

Reply:
yeah, it's a bit crazy, but AFAIK the most elegant way to compare versions in bash...

Reply:
I did not even know about `-V` in `sort`, very smart! -)

Reply:
Actually, I did find it on Stack overflow myself 😃